proof.go

```diff
@@ -71,10 +71,10 @@ func (c Claims) Valid() error {
 	if c.App == "" {
 		return fmt.Errorf("claims: app is empty")
 	}
-	if c.IssuedAt > now+drift || c.IssuedAt < now-max {
-		return fmt.Errorf("claims: iat is invalid")
+	if c.IssuedAt > now+drift {
+		return fmt.Errorf("Claim is from the future. User likely needs to sync their device's clock.")
```

Comment:
Go convention for errors is to downcase the test, can you update line to..

`return fmt.Errorf("claims: proof is issued from the future - check issuing system block.")`